test/parallel/test-http-response-cork-end.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const http = require('http');
+
+let serverRes;
+
+const server = http.createServer(common.mustCall((req, res) => {
+  serverRes = res;
+  res.writeHead(200);
+  res.write('hello ');
+
+  setImmediate(() => {
+    res.cork();
+  });
+}));
+
+server.listen(0, (err) => {
+  assert.ifError(err);
+
+  const req = http.request(server.address());
+  req.end();
+
+  req.on('response', (res) => {
+    assert.strictEqual(res.statusCode, 200);
+
+    res.setEncoding('utf8');
+    res.once('data', (chunk) => {
+      assert.strictEqual(chunk, 'hello ');
+    });
+
+    setTimeout(() => {
+      assert(serverRes);
+
+      res.once('data', (data) => {
```

Comment:
ditto :-) ... I'll stop pointing these out... just need a few more `common.mustCall()` wrappers in this and the other test cases.